target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/rtl930x.c

```diff
@@ -2312,6 +2323,12 @@ static void rtl930x_led_init(struct rtl838x_switch_priv *priv)
 	/* Set LED mode to serial (0x1) */
 	sw_w32_mask(0x3, 0x1, RTL930X_LED_GLB_CTRL);
 
+	/* Enable GPIO19_LEDSYNC if utilised */
+	if (of_property_read_bool(node, "led-sync"))
```

Comment:
We are currently using "led_set" (underscore) no need for this deviation. Only if you fix that later.

Reply:
> We are currently using "led_set" (underscore) no need for this deviation. Only if you fix that later.

My understanding was that the linux kernel prefers kebab-case property names in the device tree, and this `led-sync` aligns with the `active-low` below
```c
if (of_property_read_bool(node, "active-low"))
```